package.json

```diff
@@ -35,7 +36,7 @@
     "precommit": "lint-staged && yarn test",
     "prepare": "yarn gitbook:install",
     "prepr": "yarn test:e2e && node scripts/prePr.js --conventional-commits --yes",
-    "prepush": "npm-run-all --parallel build test build-docs:*",
+    "prepush": "npm-run-all --parallel build test build-docs:* && yarn contributors:update",
```

Comment:
I'll approve this as this is minor, but we are using `npm-run-all` here which will run all following scripts in parallel. So this should be:
`"prepush": "npm-run-all --parallel build test build-docs:* contributors:update"`

QA can make this change.